src/objective-c/ProtoRPC/ProtoService.m

```diff
@@ -65,6 +65,8 @@ - (instancetype)initWithHost:(NSString *)host
   return self;
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wincompatible-pointer-types"
```

Comment:
@makdharma's justified comment was about changing the signature of this method. We're free to change the implementation.